src/config_handler.rs

```diff
@@ -69,13 +74,19 @@ impl Config {
             wallet_address: None,
             max_capacity: None,
             root_dir: None,
+            verbose: 0,
```

Comment:
Maybe use a noun instead of an adjective? I.e.
```suggestion
            verbosity: 0,
```

Reply:
When passing the full flag I think `safe_vault --verbose` sounds slightly better than `safe_vault --verbosity`, but I'm no English speaker :)

Reply:
Ah yes, for the CLI `--verbose` works better for sure. I meant more like a structure field, but I have no strong preference here.

Reply:
Yes as a structure field I agree. `structopt` wants the field the same as the flag I think though

Reply:
That's what we can do as it seems:
https://github.com/rust-cli/clap-verbosity-flag/blob/ad073540fc6714e0d0bb186eef44eb5c1adf0c73/src/lib.rs#L35-L36

I.e.,
```rust
#[structopt(long = "verbose", short = "v", parse(from_occurrences))]
verbosity: u8,
```

Reply:
Ah yes, good point. I'll update

Reply:
I started changing but the smoke test https://github.com/maidsafe/safe_vault/pull/829/files#diff-0fb07a438eb34eab8647cbaa5b5a7482R293 doesn't seem to happy about it, so I'm inclined to keep as is?

Reply:
Happy to keep it as it is now :+1: Thanks!